config/application.rb

```diff
@@ -26,5 +26,8 @@ class Application < Rails::Application
 
     # Load vaccine configuration
     config.vaccine_standards = YAML.safe_load(File.read('config/vaccines.yml'))
+
+    # Load API configuration
+    config.api = YAML.safe_load(File.read('config/api.yml'))
```

Comment:
```suggestion
    config.api = YAML.safe_load(File.read(Rails.root.join('config', 'api.yml')))
```

Reply:
Addressed in https://github.com/SaraAlert/SaraAlert/pull/987/commits/fef5e9e9487bf3f37b213141467c2b2ae42fff10.